theforeman.org/pipelines/lib/packaging.groovy

```diff
@@ -231,17 +231,9 @@ def setup_sources_core(project, os, version, repoowner, pull_request = false) {
     dir(build_dir) {
         if (version == 'nightly') {
             def build_vars = read_build_vars(project)
-            def url_base = 'https://ci.theforeman.org'
-            def job_id = 'lastSuccessfulBuild'
-            def job_url = "${url_base}/job/${build_vars['source_location']}/${job_id}"
-
-            sh """
-              wget "${job_url}/artifact/*zip*/archive.zip"
-              unzip archive.zip
-              mv archive/pkg/*bz2 ${project}_${package_version}.orig.tar.bz2
-            """
-
-            last_commit = sh(returnStdout: true, script: "curl \"${job_url}/api/json\" | jq -r '.actions[].lastBuiltRevision.SHA1 | values'").trim()
+            copyArtifacts(projectName: build_vars['source_location'], excludes: 'package-lock.json')
```

Comment:
I didn't know if I could use `pattern` with a comma to separate. It's supposed to be the ant file pattern, but that wasn't very clear to me. This at least filters out the 1.5 MB package-lock.json file from Foreman's artifacts. Bundler's lock file is really tiny so that doesn't matter much.